admin/admin.go

```diff
@@ -230,15 +229,15 @@ func (a Admin) GetFormattedEnv(env string, reply *string) (err error) {
 	return nil
 }
 
-// StartServer starts an http server listening on unix socket and serving rpc communication
+// StartServer starts an HTTP server listening on unix socket and serving rpc communication
 func StartServer(ctx context.Context) error {
 	tmpDirPath, err := misc.CreateTMPDIR()
 	if err != nil {
 		panic(err)
 	}
 	sockAddr := filepath.Join(tmpDirPath, "rudder-server.sock")
 	if err := os.RemoveAll(sockAddr); err != nil {
-		pkgLogger.Fatal(err)
+		pkgLogger.Fatal(err) // @TODO return?
```

Comment:
We can extend the scope of this one:  https://www.notion.so/rudderstacks/rudder-server-use-context-cancellation-instead-of-panic-to-handle-failures-during-server-startup-cf5d05ce758643f68eebe204eede37ce

cc @chrikar